Nukie Aliases and Some Hairspray

Rhials opened the pull request:
## About The Pull Request

Nuclear operatives now receive an alias for their first name, instead of a default human name. This alias is randomly selected, or can be set in your name preferences.

![image](https://github.com/tgstation/tgstation/assets/28870487/2584b625-4668-4738-a662-8205476fac2f)

This does not affect Operative Leaders selecting the team's last name. This means that **operative names are now "[Alias chosen by player] [Team name chosen by leader]".**

There is a list of default aliases, in case no alias has been selected. I randomly came up with a bunch of stuff from the top of my head and put it in a text file. If you think of something better, choose it yourself!

**OH, ALSO:**

There's some hair dye in the Firebase dorms now. Useful if you want to further differentiate between your Operative/Crew characters, or just look pretty.
## Why It's Good For The Game

No longer shall nukies be "your character but with a different name". Now you can give yourself a bit more identity instead of just a random name.

More customization options means more opportunities for people doing interesting stuff, and to branch away from the beaten path of boring meta strats. 
## Changelog
:cl: Rhials
add: Nuclear Operatives, in an attempt to appeal to the more "tacticool" members of their cause, have begun using callsigns to designate themselves. Check your preferences to set your Operative Alias!
qol: At the request of the more vain members of the cause, hair dye has been added to the Operative Firebase dorms.
/:cl:

Rhials committed:
groundwork

Rhials committed:
adds hair spray to nookie base (wow)

Rhials committed:
name list and other name listing lol

Rhials committed:
makes the names work

Rhials committed:
Revert "makes the names work"

This reverts commit 79204b9f75a7e15fe8e952a97900106df45cc523.

Rhials committed:
Revert "Revert "makes the names work""

This reverts commit 2d40baeb661196803a1038b5c353aa4bbce92c8e.

Rhials committed:
ok no idea how those pipe things got changed we're just gona figure that out later ok

Rhials committed:
more names

Rhials committed:
undo!

Rhials committed:
axes leader aliases

in addition to causing problems with the alias system, this is also basically useless. The chosen name would only exist for as long as the "choose a team name" dialogue box is open, since no input will just choose a random name to override it anyways

Rhials committed:
wa la

Rhials committed:
sort

MrMelbert commented on code/modules/mob/living/carbon/human/_species.dm:
```diff
@@ -256,20 +256,24 @@ GLOBAL_LIST_EMPTY(features_by_species)
+ * * first_name - Do we use a given first name or pick a random new one?
  */
-/datum/species/proc/random_name(gender,unique,lastname)
+/datum/species/proc/random_name(gender, unique, last_name, first_name)
```
Please update the children proc with the new arg you added (In fact this will probably not work for species which implement their own proc like lizards)

Loafin34 commented:
so... fvcking... zased.....

CacheAtWork commented:
Can we get some spraytan added to, so we can nuke with a chad-like orange glow.

Rhials committed:
moves away from the random name

y'know when you really think about it, passing both a first/last name into a random name proc and ensuring nothing is actually made random is kinda wack

Rhials committed:
Merge remote-tracking branch 'upstream/master' into nukie-alias

Ghommie committed:
Merge branch 'master' into nukie-alias

Ghommie reviewed (approved):
Looks good to me, no flagrant issue that I can see.
